demo-extended/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,11 @@
       </intent-filter>
     </activity>
 
+    <receiver
+      android:name=".ShareBroadcastReceiver"
```

Comment:
I also like to be very specific on the `AndroidManifest` and that's why I would place fully qualified name here as well